### PR TITLE
RK-1355 - Update serverless build instructions in deployment examples

### PR DESCRIPTION
**Pull request by ManuelNemirovsky**

### CODE REVIEW

File: python-kubernetes/rookout-service/README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This is a full example for deploying an application with Rookout using Kubernetes.
 
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/sdk-setup.html) guide.
```
Comment (MickaelAlliel) [inline python-kubernetes/rookout-service/README.md]: ? I updated it a week ago

File: python-kubernetes/with-agent/README.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
 This is a full example for deploying an application with Rookout using Kubernetes.
 Here we are using a local ETL Agent deployed in the cluster that the SDK will connect to.
 
+Before following this guide we recommend reading the basic [Python + Rookout](https://docs.rookout.com/docs/sdk-setup.html) guide.
```
Comment (MickaelAlliel) [inline python-kubernetes/with-agent/README.md]: ? I updated it a week ago

File: python-kubernetes/rookout-service/README.md (hunk 1 above)
Comment (MickaelAlliel) [reply python-kubernetes/rookout-service/README.md]: + What does it have to do with serverless?

File: node-serverless-cross-platform/README.md [hunk 3]
```diff
@@ -1,18 +1,21 @@
-# Introduction
+# Quickstart for debugging Node + Serverless Cross Platform
+## Introduction
 Rookout has native dependencies that are compiled specifically for the current OS when installing the Rookout package specifically.  
 If your machine isn't compatible with the target app OS, the native dependencies might not work.  
 Thanks to Docker, we can easily emulate installing dependencies on a linux machine and therefore match the deploying OS with the target OS  
 
-# The App
+This sample may be out of date. If you face any issues, please reach out to mailto:support@rookout.com and let us know.
```
Comment (MickaelAlliel) [inline node-serverless-cross-platform/README.md]: ?

File: python-kubernetes/rookout-service/README.md (hunk 1 above)
Comment (ManuelNemirovsky) [reply python-kubernetes/rookout-service/README.md]: It doesn't 
I just noticed that we don't have this preview in this example, though we have it in all the others.

File: python-kubernetes/with-agent/README.md (hunk 2 above)
Comment (ManuelNemirovsky) [reply python-kubernetes/with-agent/README.md]: I just noticed that we don't have this preview in this example, though we have it in all the others.

File: python-kubernetes/with-agent/README.md [hunk 4]
```diff
@@ -3,6 +3,10 @@
...
+
+This sample may be out of date. If you face any issues, please reach out to mailto:support@rookout.com and let us know.
```
Comment (MickaelAlliel) [inline python-kubernetes/with-agent/README.md]: it was updated a week ago, @Rookeret73 should it be written it all examples?

**Pull request merged**